Field_Simulation/Simulation/WindowsFormsApp1/Form1.cs

```diff
@@ -0,0 +1,189 @@
+                        
+                        //Printing the vertical vector to the circle
+                        Invoke((MethodInvoker)delegate () { Line(circ,(vector+new Point(circ)).coordinates, 1,Color.Blue); });
+                        Invoke((MethodInvoker)delegate () { pictureBox1.Image = this.bitmap; });
+
+                        //Printing the circle
+                        
+                        Invoke((MethodInvoker)delegate () { draw_rect(circ, 2, Color.Black); });
+                        Thread.Sleep(100);
+                        }
+                    }
+                    catch(Exception exception) {
+                        MessageBox.Show(exception.Message);
+                    }
+            });
+            t.Start();
+        }
+        
+        public double get_ratio()
+        {
+            DateTime time = DateTime.Now;
+            TimeSpan t = time - this.time;
+            return (t.TotalSeconds%CIRCLE_TIME) / CIRCLE_TIME;
+
+        }
+        /// <summary>
+        /// Returns 2D vector of Circular Magnetic Field
+        /// </summary>
+        /// <param name="angle">The angle of the object relative to the path(degrees)</param>
+        /// <returns>integer array that represents the vector of the field for the specified angle</returns>
+        public int[] circular_field(double angle)
+        {
+            double a = Math.Sin(angle);
+            double b = Math.Cos(angle);
+            double x, y;
+            if(b!=0)
+            {
+                int sign = (int)(b / Math.Abs(b));
+                x = 1 * sign;
+                y = -a / b * sign;
+                double length = Math.Sqrt(x * x + y * y);
+                x = x / length;
+                y = y / length;
+                y = y * FIELD_SIZE;
+                x = x * FIELD_SIZE;
+            }
+            else
+            {
+                x = 0;
+                y = -a;
+            }
+            int[] arr = { (int)Math.Round(x), (int)Math.Round(y) };
+            return arr;
+
+        }
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="angle"></param>
+        /// <returns></returns>
+        public int[] DePolarField(double angle)
+        {
+            //TODO : fix function to work
+            return circular_field(angle * 2);
+        }
+        public double degrees_to_radians(double degrees)
+        {
+            return degrees*Math.PI/180;
+        }
+        //cleans all points from the clean list
+        public void Clean()
+        {
+            cleanM.WaitOne();
+            foreach(int[] pos in toClean)
+            {
+                bitmap.SetPixel(pos[0], pos[1], Color.White);
+            }
+            cleanM.ReleaseMutex();
+        }
+        public void draw_circle(int thickness)
+        {
+
+        }
+        public void draw_rect(int[] location,int size,Color c ,bool clean = false)
```

Comment:
Naming convention (Please use PascalCase for method names).

Operator spacing,